client/src/app/site/modules/user-components/components/user-detail-view/user-detail-view.component.ts

```diff
@@ -118,6 +118,9 @@ export class UserDetailViewComponent extends BaseUiComponent implements OnInit,
     @Input()
     public shouldEnableFormControlFn: (controlName: string) => boolean = () => true;
 
+    @Input()
+    public replaceGenderField = false;
```

Comment:
```suggestion
    public disableGenderField = false;
```

IMHO replace does not really represent what this does.